From: Jeremy Bicha <jeremy.bicha@canonical.com>
Date: Fri, 13 Jan 2023 11:38:19 -0500
Subject: Revert "firmware-security: Hide the panel when chassis type is an
 empty string"

This reverts commit 0245329949957e7f7ab56f240faaf336408ced86.

It seems to not be working correctly in some cases:
https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/2301
---
 panels/firmware-security/cc-firmware-security-panel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panels/firmware-security/cc-firmware-security-panel.c b/panels/firmware-security/cc-firmware-security-panel.c
index af75682..f7b2e79 100644
--- a/panels/firmware-security/cc-firmware-security-panel.c
+++ b/panels/firmware-security/cc-firmware-security-panel.c
@@ -638,7 +638,7 @@ update_panel_visibility (const gchar *chassis_type)
   gboolean visible = TRUE;
 
   /* there's no point showing this */
-  if (g_strcmp0 (chassis_type, "vm") == 0 || g_strcmp0 (chassis_type, "") == 0)
+  if (g_strcmp0 (chassis_type, "vm") == 0)
     visible = FALSE;
   application = CC_APPLICATION (g_application_get_default ());
   cc_shell_model_set_panel_visibility (cc_application_get_model (application),
